dts/arm/atmel/samc21.dtsi

```diff
@@ -32,6 +32,19 @@
 			prescaler = <4>;
 		};
 
+		dac0: dac@42005400 {
```

Comment:
nodes are organized by address.
the devicetree change must be extract from this commit in another one.
add the dac alias.